config

```diff
@@ -27,3 +25,4 @@ export RH_CHE_DOCKER_IMAGE_VERSION=${RH_CHE_DOCKER_IMAGE_VERSION:-${DEFAULT_RH_C
 export DOCKER_HUB_USER=${DOCKER_HUB_USER:-${RHCHEBOT_DOCKER_HUB_USER}}
 export DOCKER_HUB_PASSWORD=${DOCKER_HUB_PASSWORD:-${RHCHEBOT_DOCKER_HUB_PASSWORD}}
 export DOCKER_HUB_NAMESPACE=${DOCKER_HUB_NAMESPACE:-${RH_CHE_DOCKER_HUB_NAMESPACE}}
+export DOCKER_HUB_REGISTRY_PREFIX=${DOCKER_HUB_REGISTRY_PREFIX:-""}
```

Comment:
Can't you just set `DOCKER_HUB_NAMESPACE` in this case? 
For example `export DOCKER_HUB_NAMESPACE=docker.io/rhchestage`